library/src/main/java/com/google/android/material/motion/runtime/Scheduler.java

```diff
@@ -172,6 +172,38 @@ public void addPlan(Plan plan, Object target) {
     getTargetScope(target).commitPlan(planInfo);
   }
 
+  /**
+   * Adds a {@link NamedPlan} to this scheduler.
```

Comment:
it's important to document the side effect that a prior plan with the same name will be first removed.